cifonauta/settings.py

```diff
@@ -96,6 +96,7 @@
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'cebimar',
         'USER': 'nelas',
+        'PASSWORD': '123',
```

Comment:
This setting is only for your development environment, so you need to undo this change and be careful not to commit it again.